1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -13,8 +13,8 @@ let people = [
   { name: "Elise", age: 35 },
 ];
 
-let peopleNames =
-  /* todo */
+let peopleNames = people.map((person) => person.name);
```

Comment:
When you use the short way of writing function, if you have only one parameter, you don't need to put that in parenthesis.

.map(**person** => person.name)